src/wp-includes/abilities-api.php

```diff
@@ -1,8 +1,168 @@
 <?php
 /**
- * Abilities API
+ * Abilities API: core functions for registering and managing abilities.
+ *
+ * The Abilities API provides a unified, extensible framework for registering
+ * and executing discrete capabilities within WordPress. An "ability" is a
+ * self-contained unit of functionality with defined inputs, outputs, permissions,
+ * and execution logic.
```

Comment:
This is good! I think we need to be careful of the phrase "executing discrete capabilities..." because "capabilities" are an authentication thing in WP. I think we should stick to abilities as "functions on steroids" because, well, that's basically what they are. 😆 

```suggestion
 * The Abilities API provides a unified, extensible framework for registering
 * and executing discrete functions within WordPress. An "ability" is a
 * self-contained unit of functionality with defined inputs, outputs, permissions,
 * and execution logic.
```